NewForce-Cohort-4/tabloid-full-stack-team-magenta
# Tf view tags

# Description
​As an admin I would like to see all the available Tags so that I can choose to edit or delete one, or see that I should add a new one. Given an admin is successfully logged into the app they will locate the navigation bar and find the `tags` option. Once clicked the user should be presented with a alphabetically listed view of all tags in the database. 
​
## Type of change

# How Has This Been Tested?

1. Pull down and checkout my branch tf-view-tags
2. Start the application in VS and run the program
3. When the application loads you will also need to open the client-side portion of the project and run `npm start` from the terminal
4.  use foo@bar.com and password: password to log in to the application 
5. Once logged in navigate to the `tags` tab and click the link shown
6. From the new page ensure all tags are listed alphabetically to show the ticket was completed as required

​
# Checklist:
​
